docs/cloud/ops/restore.md

```diff
@@ -0,0 +1,13 @@
+# Restore
+
+To restore a backup, navigate to the `Backups` section fo the [Cloud Console](https://console.eventstore.cloud/) and click on the `Restore` icon of the backup you want to restore.
+
+::: card
+![one off restore backup](./images/one_off_restore.png)
+:::
+
+You will be then redirected to the usual provisioning page where you can choose your cluster topology
```

Comment:
provisioning page, where you

Reply:
topology:

Reply:
Fixed in https://github.com/EventStore/documentation/pull/309/commits/a60d8e8d917d04f339f43d47322231763e0d19dd.